openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/TestParameters.java

```diff
@@ -66,4 +68,35 @@ public static Map<String, String> getDefaultParameters() {
         }
         return mapParams;
     }
+
+    public static Map<String, String> getDefaultFairGBMParameters() {
```

Comment:
Create instead a private `getDefaultParameters` that receives the params (e.g. FairGBMDescriptorUtils.PARAMS)

and then you can have 2 methods that simply call that function, one for lgbm, another for fgbm.